modules/videopress/class.videopress-gutenberg.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+class VideoPress_Gutenberg {
+
+	/**
+	 * Initialize the VideoPress Gutenberg extension
+	 */
+	public static function init() {
+		if ( self::should_initialize() ) {
+			add_action( 'init', array( __CLASS__, 'register_video_block_with_videopress' ) );
+		}
```

Comment:
pardon me for not knowing Jetpack patterns as much, but why are we calling `should_initialize()` from inside `init()`? is there a reason not to include the `should_initialize()` logic here in this function and then conditionally load the action if the conditions hold?

Reply:
I only did this in order to follow the same pattern I observed in the `Jetpack_Gutenberg` class: https://github.com/Automattic/jetpack/blob/535f34b30532ea1fe39934f1201ac760b98ca447/class.jetpack-gutenberg.php#L174

Reply:
let's see if @roccotripaldi has any thoughts here. it looks like this may have been [done in a refactor for the sake of reusing a common condition](https://github.com/Automattic/jetpack/pull/10893) among many places. if we don't have a strong reason for copying that here how about we just remove the layer and keep it straightforward in `init`?

Reply:
After confirming that only the Gutenberg check was the only one really needed (this code is never executed if Jetpack is inactive and the module is always loaded in WP.com if we have the right plan), I simplified everything, removing the `should_initialize` function and moving the Gutenberg check directly to the `init` function.